### PR TITLE
Possibly fixed a few NPEs regarding null piloted crafts

### PR DESCRIPTION
**Pull request by TylerS1066**
### Checklist
- [x] Unit tests 
- [x] Proper internationalization
- [x] Compiled/tested

### CODE REVIEW

**Comment by TylerS1066** [comment]
This possible logic issue has been patched,

**Comment by TylerS1066** [comment]
@cccm5 @eirikh1996 can one of you review this?  I'd like to get this included soon.

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncManager.java [hunk 1]
```diff
@@ -671,49 +671,50 @@ private void processDetection() {
...
+                                    notification += I18nSupport.getInternationalisedString("Contact/Subcraft Rotate - West");
+                            else if (diffz < 0)
+                                notification += I18nSupport.getInternationalisedString("Contact/Subcraft Rotate - South");
                             else
-                                notification += I18nSupport.getInternationalisedString("Contact/Subcraft Rotate - West");
-                        else if (diffz < 0)
-                            notification += I18nSupport.getInternationalisedString("Contact/Subcraft Rotate - South");
-                        else
-                            notification += I18nSupport.getInternationalisedString("Contact/Subcraft Rotate - North");
+                                notification += I18nSupport.getInternationalisedString("Contact/Subcraft Rotate - North");
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncManager.java:706]: Use brackets

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ReleaseCommand.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
         if (args[0].equalsIgnoreCase("-n")) {
             final List<Craft> craftsToRelease = new ArrayList<>(CraftManager.getInstance().getCraftList());
             for (Craft craft : craftsToRelease) {
-                if(craft.getNotificationPlayer()==null) {
+                if(craft.getNotificationPlayer() == null) {
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ReleaseCommand.java:74]: don't touch this

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/craft/ICraft.java [hunk 3]
```diff
@@ -64,7 +64,7 @@ public void translate(@NotNull World world, int dx, int dy, int dz) {
     @Override
     public void rotate(Rotation rotation, MovecraftLocation originPoint) {
         if(getLastRotateTime()+1e9>System.nanoTime()){
-            if(getNotificationPlayer()!=null)
+            if(getNotificationPlayer() != null)
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/craft/ICraft.java:67]: don't touch this

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/worldguard/WorldGuardCompatManager.java [hunk 4]
```diff
@@ -32,7 +32,7 @@ public void onCraftRotateEvent(CraftRotateEvent event){
         if(event.getCraft().getNotificationPlayer() == null)
             return;
         for(MovecraftLocation location : event.getNewHitBox()){
-            if(!Movecraft.getInstance().getWorldGuardPlugin().canBuild(event.getCraft().getNotificationPlayer(),location.toBukkit(event.getCraft().getW()))){
+            if(!Movecraft.getInstance().getWorldGuardPlugin().canBuild(event.getCraft().getNotificationPlayer(), location.toBukkit(event.getCraft().getW()))){
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/worldguard/WorldGuardCompatManager.java:35]: don't touch this

**Review by cccm5 — CHANGES_REQUESTED** [review]

**Review by cccm5 — APPROVED** [review]

**Pull request merged**